src/Version10/ExecPluginInterface.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Phpcq\PluginApi\Version10;
+
+use Phpcq\PluginApi\Version10\Definition\ExecTaskDefinitionBuilderInterface;
+use Phpcq\PluginApi\Version10\Task\OutputWritingTaskInterface;
+use Phpcq\PluginApi\Version10\Task\TaskInterface;
+
+/**
+ * This interface describes plugins providing executable tasks.
+ */
+interface ExecPluginInterface extends PluginInterface
+{
+    /**
+     * Describe the exec task which is provided by the plugin.
+     *
+     * @param ExecTaskDefinitionBuilderInterface $definitionBuilder The exec task definition builder.
+     */
+    public function describeExecTask(
+        ExecTaskDefinitionBuilderInterface $definitionBuilder,
+        EnvironmentInterface $environment
```

Comment:
I think we agreed that in the interfaces all parameters must be documented with their purpose.
Environment is missing here.